tests/e2e/e2e_common.go

```diff
@@ -204,6 +204,7 @@ const (
 	topologyCluster                            = "TOPOLOGY_CLUSTERS"
 	topologyLength                             = 5
 	tkgshaTopologyLevels                       = 1
+	envK8sVersion                              = "K8S_VERSION"
```

Comment:
you can use `client.Discovery().ServerVersion()`

Reply:
done